R/cv_varsel.R

```diff
@@ -1266,7 +1314,7 @@ kfold_varsel <- function(refmodel, method, nterms_max, ndraws, nclusters,
     } else if (!search_out_rks_was_null) {
       search_path <- list(predictor_ranking = rk)
     } else {
-      search_path <- select(
+      search_path <- .select(
```

Comment:
Was there a name conflict that caused this renaming of `select()`? I think renaming this function is a good idea, but personally, I prefer function names not starting with a dot because that "hides" those function names (see [here](https://stackoverflow.com/questions/7526467/what-does-the-dot-mean-in-r-personal-preference-naming-convention-or-more/7526926#7526926), for example).

Reply:
I did get a name conflict while testing, and in addition it was confusing when reading the code. I used dot as I had seen it used in other Stan related packages. I'm fine also with some other name if you have suggestions